pkg/parser/epub/metadata.go

```diff
@@ -674,13 +699,32 @@ func (m PubMetadataAdapter) TDM() *manifest.TDM {
 	return tdm
 }
 
-func (m PubMetadataAdapter) a11yConformsTo() []manifest.A11yProfile {
+func (m PubMetadataAdapter) a11yConformsTo() ([]manifest.A11yProfile, manifest.A11y) {
 	profiles := manifest.A11yProfileList{}
+	a11y := manifest.NewA11y()
 
-	if items, ok := m.items[VocabularyDCTerms+"conformsto"]; ok {
+	if items, ok := m.items[VocabularyDCTerms+"conformsTo"]; ok {
 		for _, item := range items {
 			if profile := a11yProfile(item.value); profile != "" {
 				profiles = append(profiles, profile)
+				for k, v := range item.children {
```

Comment:
Do I read correctly that the refined metadata of the last `conformsTo` declared will override all the other accessibility metadata? In which case we're loosing the other metadata (global or refined). Is this what we want @HadrienGardeur?

Reply:
I'm not sure I'm following that one since `conformsTo` should be an array of URIs.

Reply:
If I understand the code correctly this will parse the a11y metadata that are refining each `dc:conformsTo` tags. Although I didn't know that `<dc:conformsTo>` could be refined, I thought only `<meta>` with an ID could.

@chocolatkey Do you have an OPF example that this is supposed to parse?

Reply:
This is the example that we were using: https://github.com/standardebooks/nathaniel-hawthorne_the-house-of-the-seven-gables/blob/master/src/epub/content.opf#L19

Reply:
Ha I see, in that case my initial question stands. We can have several conformances with different IDs and their own refinements.

```xml
<meta id="conformance-statement" property="dcterms:conformsTo">EPUB Accessibility 1.1 - WCAG 2.2 Level AA</meta>
```

Reply:
Then it's probably a limitation of our model because the `certification` object isn't tied to a particular `conformsTo`. 

I'm hesitant to shake things up though, because it seems quite unlikely that you'd get certified multiple times (in most countries, there won't be a certification process at all).